ui/src/components/Investor/Investor.tsx

```diff
@@ -13,14 +12,14 @@ import {
 } from '@daml.js/da-marketplace/lib/Marketplace/Investor'
 import { MarketRole } from '@daml.js/da-marketplace/lib/Marketplace/Utils'
 
-import { wrapDamlTuple } from '../common/damlTypes'
+import { useOperator } from '../common/common'
+import { wrapDamlTuple, damlTupleToString } from '../common/damlTypes'
```

Comment:
is the `damlTupleToString` import necessary?

Reply:
yep, it gets used below to map public registry data for the exchanges an investor is a participant of. I think its showing up as an addition because I moved the import statements around